mailbox/elasticsearch/src/main/java/org/apache/james/mailbox/elasticsearch/json/IndexableMessage.java

```diff
@@ -155,11 +162,12 @@ private void generateText() {
     private Optional<String> bodyHtml;
     private String text;
 
+
     public IndexableMessage(long uid, String mailboxId, List<String> users, long modSeq, long size, String date, String mediaType,
```

Comment:
That's maybe a lot of work, but it can be cool. Try to do it as a separate commit, and don't invest too much time on it ;-)